workloads/virtual-machines/disks-and-volumes.md

```diff
@@ -376,6 +379,47 @@ spec:
 
 Ephemeral VMs very often come with read-only root images and limited tmpfs space. In many cases this is not enough to install application dependencies and provide enough disk space for the application data. While this data is not critical and thus can be lost, it is still needed for the application to function properly during its lifetime. This is where an `emptyDisk` can be useful. An emptyDisk is often used and mounted somewhere in `/var/lib` or `/var/run`.
 
+### hostDisk
+
+A ```hostDisk``` volume type provides the ability to create or use a disk image located somewhere on a node.
+It works similar to a ```hostPath``` in Kubernetes and provides two usage types:
+
+* ```DiskOrCreate``` if a disk image does not exist at a given location then create one
+* ```Disk``` a disk image must exist at a given location
+
+Example: Create a disk image with 1Gi size at /data/disk.img location and attach it to a VM.
```

Comment:
Suggest rewording to:

Example: Create a 1Gi disk image located at /data/disk.img and attach it to a VM.

Reply:
done